src/mkdocstrings/templates/python/material/attribute.html

```diff
@@ -17,6 +17,7 @@
       {% endif %}
 
       {% filter heading(heading_level,
+          role="data" if obj == module else "attr",
```

Comment:
I must add tests for this. Not sure if it would fail silently or crash when someone renders `::: something.that.isnot.a.module` and therefore the `module` variable does not exist.

Reply:
It works fine with module-level and class-level attributes, in both cases (rendering a module recursively or just a class recursively), so I'll go ahead without tests. But it would be nice to have some end-to-end rendering tests at some point.